vignettes/rhino.Rmd

```diff
@@ -0,0 +1,630 @@
+---
+title: "shiny.i18n in Rhino applications"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{shiny.i18n in Rhino applications}
+  %\VignetteEngine{knitr::knitr}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Introduction
+
+This tutorial shows an easy way of implementing internationalization in a Shiny application
+built using [Rhino framework](https://appsilon.github.io/rhino/).
+The goal is to recreate a simple app that can be [found in the
+examples](https://github.com/Appsilon/shiny.i18n/blob/master/examples/live_language_change/browser_app.R), but with the Rhino setup.
+
+# Setup Rhino application
+
+The first step is to create a Rhino application.
+Before that, make sure that you have `rhino` installed:
+
+```r
+install.packages("rhino")
+```
+
+Now, initialize a new Rhino application. You can do it either by using RStudio Wizard
+or a function `rhino::init("rhino_i18n")`.
+More details on how to create a Rhino app can be found in the [Rhino tutorial](https://appsilon.github.io/rhino/articles/tutorial/create-your-first-rhino-app.html#create-an-initial-application).
+
+After setting the initial application, the next step is to install `shiny.i18n` and add it to the dependencies.
+
+_NOTE: `shiny.i18n` compatibility with `rhino` is currently available only in the development version. The installation guide will be updated after the CRAN release._
+
+In the R console run:
+```r
+renv::install("remotes")
+remotes::install_github("Appsilon/shiny.i18n")
```

Comment:
Compatibility with Rhino is available only in the dev version. Since the CRAN release is blocked due to a problem with access to the previous maintainer email, this needs to point to the GitHub version.